week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -8,9 +8,15 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
+  constructor(){
+    this.shoppingBasket = []
+  }// Add your code here
 
+  addItem(item){
+    this.shoppingBasket.push(item);
+  }
   cartContains() {
+    console.log(this.shoppingBasket)
```

Comment:
Make sure to pay attention to what you are being asked (The output of running your code should be:
 "Your shopping cart has 3 items: Toilet Roll, Pasta, Eggs") 👀